rspec-sidekiq.gemspec

```diff
@@ -12,8 +12,11 @@ Gem::Specification.new do |s|
   s.license     = 'MIT'
 
   s.add_dependency 'rspec-core', '~> 3.0', '>= 3.0.0'
-  s.add_dependency 'sidekiq', '>= 2.4.0'
+  s.add_dependency 'sidekiq', '>= 2.4.0', '< 7'
```

Comment:
📓 This is temporary while Sidekiq 7 support is being sorted out, I expect Sidekiq 7 to still be supported - but I'll have to tweak the specs for `be_delayed` or drop it entirely (since it's gone from Sidekiq 7)